napari/_app_model/constants/_commands.py

```diff
@@ -77,7 +77,7 @@ class CommandId(str, Enum):
     LAYER_PROJECT_MEDIAN = 'napari:layer:project_median'
 
     @property
-    def title(self) -> str:
+    def title(self) -> str: # type: ignore[override]
```

Comment:
str, already have a title method, so we're shadowing it with a property which is technically incompatible. So ignore.